src/main/java/com/myroslav/cosmickitties/controller/ProductController.java

```diff
@@ -0,0 +1,49 @@
+package com.myroslav.cosmickitties.controller;
+
+import com.myroslav.cosmickitties.dto.ProductDTO;
+import com.myroslav.cosmickitties.service.ProductService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/products")
```

Comment:
Please checkout the lecture - we talked about this -1
<img width="572" height="437" alt="image" src="https://github.com/user-attachments/assets/ac30d8ac-944f-4f56-b912-7aea85d3af40" />